src/System.Private.Uri/src/System/IriHelper.cs

```diff
@@ -198,7 +198,7 @@ internal static unsafe string EscapeUnescapeIri(char* pInput, int start, int end
                             int startSeq = next;
                             int byteCount = 1;
                             // lazy initialization of max size, will reuse the array for next sequences
-                            if ((object)bytes == null)
+                            if ((object?)bytes == null)
```

Comment:
> Why not just fix this?

Because it shows up in a ton of places and I don't want to mix that in with the minimum necessary to add the annotations.  I'd be fine with someone submitting a subsequent PR to clean these up.